### DISCUSSION TITLE
How can i increase retry count for sepecific error

### DISCUSSION THREAD

**Discussion by zedongh** [uber/cadence#4659] [Q&A] [answered]
Sometimes, some specific errors can be solved by get more retry times, but some errors not.

**Comment by zedongh** [comment, marked as answer]
Finally, I setup a retry policy with large retry count and no retry reasons for my workflow, and worker control retry count and custom error solved

**Reply by longquanzheng** [reply]
glad to know that you figured it out. Cadence doesn't have that support out of the box. And generally it's not recommended to have such logic. Either some errors can be retry, not retry.
